front/src/config/i18n/en.json

```diff
@@ -863,6 +863,52 @@
         "saveLabel": "Save configuration"
       }
     },
+    "oauth2": {
```

Comment:
I can change this but I put those keys in integration path because this component is useful only for on integration. I fear if I put those keys in common part it's confuse,
what you prefer ?

Reply:
Usually we only put integration translations in this part. I would prefer to move at root level yes.
I pretend we have a "oauth2" integration, but it is a common component/feature.

Reply:
Ok I put it in root level